Makefile

```diff
@@ -12,3 +12,4 @@ flake8:
 
 test: flake8
 	nosetests openfisca_web_api --exe --with-doctest
+	bash openfisca_web_api/tests/test-cli.sh
```

Comment:
Do you really need to specify `bash`? It's better to rely on [`sh` rather than a specific implementation](http://stackoverflow.com/questions/5725296/difference-between-sh-and-bash).
Make the file executable and leave it to the host.